sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TelemetryExceptionDataTests.cs

```diff
@@ -232,7 +232,13 @@ public void ExceptionDataContainsExceptionDetailsofAllInnerExceptionsOfAggregate
             var exceptionData = new TelemetryExceptionData(2, logRecords[0]);
 
             Assert.Equal(3, exceptionData.Exceptions.Count);
+
+#if NET461
             Assert.Equal("AggregateException", exceptionData.Exceptions[0].Message);
+#else
+            Assert.Equal("AggregateException (Inner1) (Inner2)", exceptionData.Exceptions[0].Message);
```

Comment:
Could we jut assert on `aggregateException.Message`?

Reply:
Yes! that's much clearer. Thanks!!